openhtf/io/output/print_console.py

```diff
@@ -0,0 +1,81 @@
+"""Module for outputting test record to JSON-formatted files."""
+import os
+import sys
+from json import JSONEncoder
+from openhtf import util
+
+
+class PrintConsole(JSONEncoder):
```

Comment:
hm, i don't think you have to inherit from JSONEncoder, it doesn't look like you are using it anywhere?